docs/UserDoc_Chap5_Create.rst

```diff
@@ -57,17 +57,19 @@ Basic templates are available for users to create metadata directly on `spatiald
 
 To create a metadata record using an available template:
 
-	**1|** Click the |button_contribute| button in the header menu. This will enable the **Contribute** board, where all records the user has access to will be listed.
+	**1|** Click the |button_contribute| button in the header menu. 
 	
-	**2|** Click the |button_contribute_addrecord| button.
+	**2|** Click the |button_editor_board| button. This will enable the **Contribute** board, where all records the user has access to will be listed.

```

Comment:
I think when you select the Contribute tab, the actual board is called the Editor board. However changing this may mean changing it elsewhere

Reply:
done